server/JWT.js

```diff
@@ -27,7 +27,7 @@ let JWT = {
       var parts = req.headers.authorization.split(' ')
       if (parts.length === 2 && /^Bearer$/i.test(parts[0])) { return parts[1] }
       throw new Error('Format is Authorization: Bearer [token]')
-    } else if (req.param('token')) {
+    } else if (req.body.hasOwnProperty('token')) {
```

Comment:
The token can also come on the query - which is why we are deleting req.query.token as well.